layouts/rss.xml

```diff
@@ -11,16 +11,12 @@
 {{- $pages = $pages | first $limit -}}
 {{- end -}}
 {{- printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\"?>" | safeHTML }}
-<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/">
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:webfeeds="http://webfeeds.org/rss/1.0">
```

Comment:
You should keep both the `webfeeds` and `content` since you are also using `content` in your latest revision.

Reply:
Okay, I added the content part and push a commit. Thank you so much for your patience.